lametro/templates/board_members/board_members.html

```diff
@@ -18,134 +18,142 @@
+
+            <div class="col-lg-6 px-0 px-lg-3">
+                <div id="council-member-scroll-area">
                     {% include 'board_members/_council_member_table.html' %}
                 </div>
             </div>
         {% else %}
-            <div class="col-sm-8 no-pad-mobile">
+            <div id="council-member-scroll-area" class="col-8 px-0 px-md-3">
                 {% include 'board_members/_council_member_table.html' %}
             </div>
-            <div class='col-sm-4'>
+            <div class="col-4">
                 {% include 'board_members/_council_info_blurb.html' %}
             </div>
         {% endif %}
     </div>
 </div>
-<div class="container-fluid">
-    <div class="row-fluid">
-        <div class="col-sm-8">
+
+<div class="container-fluid px-4 px-lg-5 mt-5 mt-lg-4">
+    <div class="row">
+        <div class="col-lg-8">
             <h4><i class='fa fa-fw fa-list-ul'></i> Recent Activity</h4>
-            <table class='table' id='committee-actions'>
+            <table class="table" id="committee-actions">
                 <thead>
                     <tr>
-                        <th>Action</th>
-                        <th></th>
+                        <th colspan="2">Action</th>
                     </tr>
                 </thead>
                 <tbody>
-                    {% for action in recent_activity|slice:":100" %}
-                        <tr class="activity-row">
-                            <td class='nowrap'>
-                                <p class="text-muted small no-pad-bottom">
-                                    {{action.date_dt|date:'n/d/Y'}}
-                                </p>
-                                <p class="small no-pad-bottom">
-                                    <span class='text-{{action.label}}'>{{action.description | remove_action_subj}}</span>
-                                </p>
-                            </td>
-                            <td>
-                                <p class="small no-pad-bottom">
-                                    <a href="/board-report/{{action.bill.slug}}/">{{action.bill.friendly_name}}</a>
-                                </p>
-                                <p class="small no-pad-bottom">
-                                    {{action.bill.title | short_blurb}}
-                                </p>
-                            </td>
-                        </tr>
-                    {% endfor %}
                     <tr>
-                        <td colspan="3" align="center">
+                        <td class="text-end text-lg-center" colspan="3">
                             <a href="" id="more-actions"><i class="fa fa-fw fa-chevron-down"></i>Show more</a>
                             <a href="" id="fewer-actions"><i class="fa fa-fw fa-chevron-up"></i>Show fewer</a>
                         </td>
                     </tr>
+                    {% for action in recent_activity|slice:":100" %}
```

Comment:
It might make sense to keep the "Show more" option at the bottom and just reduce the number of items shown on the page (100 is a lot of items to scroll through!).

What do you think?

Reply:
That sounds good! Maybe 25? Or is that too few?

Reply:
Sure, I think 25 is reasonable.

Reply:
Awesome, got those changes in